pkg/operator/ceph/object/status_test.go

```diff
@@ -36,4 +36,33 @@ func TestBuildStatusInfo(t *testing.T) {
 	statusInfo := buildStatusInfo(cephObjectStore)
 	assert.NotEmpty(t, statusInfo["endpoint"])
 	assert.Equal(t, "http://rook-ceph-rgw-my-store.rook-ceph:80", statusInfo["endpoint"])
+
+	cephObjectStore = &cephv1.CephObjectStore{
```

Comment:
Add a comment before the line to describe the test

Reply:
Done.